### PR TITLE
implement login, register feature

### PR DESCRIPTION
**Pull request by bdnkhanhs**

### CODE REVIEW

File: src/features/Login/index.jsx [hunk 1]
```diff
@@ -1,12 +1,67 @@
...
+    history.push("/user-register")
 
-    return (
-        <>Trang login</>
-    );
+  }
+
+  return (
+    <>
+      <div className="wrapper_Login">
```
Comment (NghiaBui486) [inline src/features/Login/index.jsx]: Using keyword lowercase for className

File: src/features/Login/index.scss [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+    background-color: #f0f2f5;
+}
+
+.wrapper_Login {
+    margin: 0px auto;
+    padding: 70px 140px 60px 140px;
+    .form_left{
+        background-color: #ffffff;
+        padding: 10px 100px 60px 30px;
+        .form_content h1{
```
Comment (NghiaBui486) [inline src/features/Login/index.scss]: using &_content... to replace .form_content

File: src/features/Login/index.scss [hunk 3]
```diff
@@ -1,4 +1,27 @@
...
-    margin: 100px auto;
-    width: 500px;
+body{
+    background-color: #f0f2f5;
+}
+
+.wrapper_Login {
+    margin: 0px auto;
+    padding: 70px 140px 60px 140px;
+    .form_left{
```
Comment (NghiaBui486) [inline src/features/Login/index.scss]: I think the class name need move out to wrapper_Login class

File: src/features/Login/index.scss [hunk 4]
```diff
@@ -1,4 +1,27 @@
...
+        background-color: #ffffff;
+        padding: 10px 100px 60px 30px;
+        .form_content h1{
+            color: #3278e4;
+            margin: 100px 20px 70px 0px;
+        }
+    }
+    .content_right{
```
Comment (NghiaBui486) [inline src/features/Login/index.scss]: fix class name like above

File: src/features/Login/index.jsx [hunk 5]
```diff
@@ -1,12 +1,67 @@
-import React, { useState } from 'react';
-import { Form, Input, Button, Checkbox } from 'antd';
-import "./index.scss"
+import React, { useState } from "react";
+import { Form, Input, Button, Row, Col, Space } from "antd";
+import { useHistory } from "react-router-dom"
+import "./index.scss";
+import logo from "../../assets/images/logo.png";
```
Comment (NghiaBui486) [inline src/features/Login/index.jsx]: Can define a file constant for image, and import to using

File: src/features/Login/index.jsx (hunk 1 above)
Comment (NghiaBui486) [reply src/features/Login/index.jsx]: Fixed at commit [fa21ab90a|https://github.com/NghiaBui486/share-room/pull/1/commits/fa21ab90a7261dcb6a6caf2f0df8eea6157205fd]

Comment (bdnkhanhs) [reply src/features/Login/index.jsx]: Fixed at commit [fd36968|https://github.com/NghiaBui486/share-room/pull/1]

File: src/features/Login/index.scss (hunk 2 above)
Comment (bdnkhanhs) [reply src/features/Login/index.scss]: Fixed at commit [fd36968|https://github.com//pull/1]

File: src/features/Login/index.scss (hunk 4 above)
Comment (bdnkhanhs) [reply src/features/Login/index.scss]: Fixed at commit [fd36968|https://github.com//pull/1]

File: src/features/Login/index.jsx (hunk 5 above)
Comment (bdnkhanhs) [reply src/features/Login/index.jsx]: Fixed at commit [fd36968|https://github.com//pull/1]

**Pull request merged**